app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :stylesheet do %>
+  <%= stylesheet_link_tag 'company', 'data-turbolinks-track' => true %>
+<% end %>
+<div class='form'>
+  <%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post, class: 'form-signin' }) do |f| %>
+
+    <% if resource.errors.any? %>
+      <div class='alert fade in alert-danger'>
+        <button class="close" data-dismiss="alert">×</button>
+        <%= resource.errors.full_messages.join(' and ') %>
```

Comment:
Let's use resource.errors.full_messages.to_sentence. It will construct a meaningful message